101-private-endpoint-webapp/azuredeploy.json

```diff
@@ -0,0 +1,224 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "virtualNetwork_name": {
```

Comment:
add metadata description for all parameters

Reply:
I added for each parameter a metadata